contracts/src/interfaces/ITeleporterTokenBridge.sol

```diff
@@ -31,6 +32,7 @@ struct SendTokensInput {
     bytes32 destinationBlockchainID;
     address destinationBridgeAddress;
     address recipient;
+    address feeTokenAddress;
```

Comment:
Nit: May be more clear to call this `primaryFeeTokenAddress` since it only applies for the primary fee